src/applications/claims-status/actions/index.js

```diff
@@ -680,6 +680,188 @@ export function submitFiles(claimId, trackedItem, files) {
   };
 }
 
+// START lighthouse_migration
+export function submitFilesLighthouse(claimId, trackedItem, files) {
+  let filesComplete = 0;
+  let bytesComplete = 0;
+  let hasError = false;
+  const totalSize = files.reduce((sum, file) => sum + file.file.size, 0);
+  const totalFiles = files.length;
+  // START lighthouse_migration
+  const trackedItemId = trackedItem ? getTrackedItemId(trackedItem) : null;
+  // END lighthouse_migration
+  recordEvent({
+    event: 'claims-upload-start',
+  });
+
+  return dispatch => {
+    dispatch(clearNotification());
+    dispatch(clearAdditionalEvidenceNotification());
+    dispatch({
+      type: SET_UPLOADING,
+      uploading: true,
+    });
+    dispatch({
+      type: SET_PROGRESS,
+      progress: 0,
+    });
+    require.ensure(
+      [],
+      require => {
+        const csrfTokenStored = localStorage.getItem('csrfToken');
+        const { FineUploaderBasic } = require('fine-uploader/lib/core');
+        const uploader = new FineUploaderBasic({
+          request: {
+            endpoint: `${
+              environment.API_URL
+            }/v0/benefits_claims/${claimId}/benefits_documents`,
```

Comment:
i might have missed something as i was comparing side-by-side so definitely push back if i did, but this line seems to be the only difference between this function and `submitFiles` that i could find — do you think it'd be possible to DRY this up a bit and adjust `submitFiles` to swap out the path depending on if the feature flag is toggled on?

Reply:
@anniebtran gonna push back on this after discussion with @jerekshoe. Despite the duplication, these two functions are used as part of a feature toggle (one gets used vs the other based on the toggle). The old one will be getting deleted, and the new one may become more complex, so we want to avoid injection feature toggle level logic inside the function to keep it clean. Appreciate the feedback!

Reply:
cool, that makes sense to me — thanks for clarifying! do you mind putting a `TODO` or comment near the old one just so anyone poking around the codebase knows that it's going to be removed in the future?